### ISSUE TITLE
`imported multiple times` error due to wrong types?

### ISSUE THREAD

**Issue by seanlail** [sveltejs/eslint-plugin-svelte#556]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.46.0

### What version of `eslint-plugin-svelte` are you using?

2.32.4

### What did you do?

In a component I have this:
```svelte
<script lang="ts">
  import { createEventDispatcher } from "svelte";
  import { fade } from "svelte/transition";
```

### What did you expect to happen?

No warning or error from ESLint

### What actually happened?

I see this error in VSCode and when I run eslint:
```bash
/Users/.../src/components/Expandable.svelte
  2:41  warning  '/Users/.../node_modules/svelte/types/index.d.ts' imported multiple times  import/no-duplicates
  3:24  warning  '/Users/.../node_modules/svelte/types/index.d.ts' imported multiple times  import/no-duplicates
```

When I save the file (I have formatOnSave enabled) it moves the `fade` import to the wrong module and breaks the build:
```svelte
<script lang="ts">
  import { createEventDispatcher, fade } from "svelte";
```

### Link to **GitHub Repo** with Minimal Reproducible Example

https://github.com/seanlail/lint-issue-556

Last commit shows the issue.
If you run this locally, remove [the comment here](https://github.com/seanlail/lint-issue-556/commit/b7d4f2d5276f163a2961c8986ee38301d1086006) and you will see the issue.

### Additional comments

npm: 9.5.0
node: v18.14.2
svelte: 4.1.2
typescript: 5.1.6

**Comment by ota-meshi** [comment]
Thank you for posting the issue. However, I think it's an issue with the `import/no-duplicates` rule. It's the same for `*.ts` files. Please post an issue in the eslint-plugin-import repository.
We can't do anything with this repository, so I'm closing this issue.

<img width="804" alt="image" src="https://github.com/sveltejs/eslint-plugin-svelte/assets/16508807/376d6bc3-e5db-40c4-9094-aa838176f737">
